tables/heb.tbl

```diff
@@ -0,0 +1,260 @@
+
+# liblouis: Hebrew
+#
+# ------------
+#-index-name: Hebrew uncontracted
+#-display-name: Hebrew uncontracted
+#
+#+language:heb
+#+type:literary
+#+contraction:no
+#+system:heb
+
+#-license: lgpl-2.1
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+# liblouis  comes with ABSOLUTELY NO WARRANTY.
+
+
+# This table represents Classical or Biblical Hebrew. This table is different from a modern Hebrew table because this table provides support for the points, vowels, cantillation marks, and for the critical apparatus for the BHS.This table only departs from the above mentioned standards, when an innovative braille code is required to represent symbols in the Critical apparatus and cantillation marks, that have never had braille assigned to them before. Modern Hebrew tables do not provide this support. for more information on how the standard for  Biblical Hebrew was developed, please see to the following links::
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
```

Comment:
So it seems a lot more background information about the braille code could be added. I thought [@IgiNuDu8 explained it very well in this comment](https://github.com/liblouis/liblouis/pull/1564#issuecomment-2115808385). Consise and clear:

> Just to chime in on Hebrew braille standards, the standard for Classical/Biblical Hebrew differs from the modern standards set by Israel's Central Library for Blind and Visually Impaired People, which is included in World Braille Usage, 3d ed. That standard is for Modern Hebrew, which is usually written without vowels (as others have said above), and does not include many of the accents specific to biblical texts. The standard for biblical Hebrew is based on the International Hebrew Braille Code (IHBC), as set out in the Preface to the JBI Braille Tanakh. This standard has some flexibility in regard to the inclusion of silent shewas, and only included those accents that served a grammatical purpose (as punctuation), not those used for chanting. Unfortunately, the IHBC did not develop standards for all of the accents and cantillation marks, and several independent systems have been developed. Matt, you can correct me if I'm wrong, but I believe the system in this table is based on the accents developed by Ray McAllister.

If you don't mind adding something like this (in Eric's words or in your own words), that would be great! Also what you mention about your own contribution to the braille code:

> You are correct about Ray McAllester writing much of the braille code for the cantillation marks. There were several that he did not write because he may have not had access to them. I wrote the rest of the cantillations that he did not, following the same pattern that he developed.